src/shared/resourcefetcher/compositeResourceFetcher.ts

```diff
@@ -0,0 +1,47 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { getLogger, Logger } from '../logger'
+import { ResourceFetcher } from './resourcefetcher'
+
+export class CompositeResourceFetcher implements ResourceFetcher {
+    private readonly logger: Logger = getLogger()
+    private readonly fetchers: ResourceFetcher[]
+
+    /**
+     * Tries loading from fetchers in order until one returns a resource
```

Comment:
Nit: this comment should probably be moved to the get function's documentation.

Reply:
why not both dot jpeg